Project.toml

```
@@ -13,11 +13,10 @@ Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
 
 [compat]
 AxisArrays = "0.4"
-Compat = "2.2, 3"
 Distributions = "0.21, 0.22, 0.23, 0.24, 0.25"
 NamedDims = "0.2, 0.3, 1"
 Tables = "1"
-julia = "1"
+julia = "1.5"
```

```suggestion
julia = "1.6"
```

1.6 is the current LTS, and I think it's reasonable to decline support for versions before that, even if there's not yet a technical reason to do that.

---

CI should be updated to remove version 1.0 and replace with the oldest version we support (1.5 or 1.6): https://github.com/invenia/ObservationDims.jl/blob/master/.github/workflows/CI.yml

The "include" for 1.5 based on Invenia can be removed as well.